test/src/test/java/jenkins/model/JenkinsLocationConfigurationTest.java

```diff
@@ -127,8 +127,9 @@ public void doNotAcceptNonHttpBasedRootURL_fromConfigXml() {
 
     @Test
     @Issue("SECURITY-1471")
-    public void cannotInjectJavaScriptUsingRootUrl_inNewViewLinkAction() throws Exception {
```

Comment:
I've updated the test as the 'New View' sidebar action is gone now. The updated test instead tests the 'New View' tab bar link. I've had to create a new project at the start of the test to bypass the 'Welcome to Jenkins!' screen.